src/router/bundle/bundleHandler.ts

```diff
@@ -146,6 +147,32 @@ export default class BundleHandler implements BundleHandlerInterface {
             }
         }
 
+        const authAndFilterReadPromises = requests.map((request, index) => {

```

Comment:
are these promises? You might need to add `async`

Reply:
https://flaviocopes.com/javascript-async-await-array-map/
"Remember, we must wrap any code that calls await in an async function."

Yep, they are promises, but since we don't initiate `await` inside of the map function, we don't have to prefix the function with `async`

Reply:
Super helpful link thanks